tests/elb/test_elb.py

```diff
@@ -190,3 +196,25 @@ def test_elb_add_listener_policy(mock_get_properties, mock_boto3_session):
     elb = SpinnakerELB(app='myapp', env='dev', region='us-east-1')
     elb.add_listener_policy(json.dumps(json_data))
     assert client.set_load_balancer_policies_of_listener.called
+
+@mock.patch('foremast.elb.create_elb.boto3.session.Session')
+@mock.patch('foremast.elb.create_elb.get_properties')
+def test_elb_add_backend_policy(mock_get_properties, mock_boto3_session):
+
+    json_data = {
+       'job': [
+            {
+                'listeners': [
+                    {
+                        'internalPort': 80,
+                        'backendPolicies': ['policy_name']
+                    },
+                ],
+            }
+       ],
+    }
+    client = mock_boto3_session.return_value.client.return_value
+
+    elb = SpinnakerELB(app='myapp', env='dev', region='us-east-1')
+    elb.add_backend_policy(json.dumps(json_data))
+    assert client.set_load_balancer_policies_for_backend_server.called
```

Comment:
Use the builtin `assert_called()` instead.
```python
client.set_load_balancer_policies_for_backend_server.assert_called()
```
It would be better to check that it was called with specific arguments.
```python
client.set_load_balancer_policies_for_backend_server.assert_called_with(LoadBalancerName='', InstancePort='', PolicyNames=[])
```
If you do not care about checking an argument, use `mock.ANY`.
```python
client.set_load_balancer_policies_for_backend_server.assert_called_with(LoadBalancerName=mock.ANY, InstancePort=mock.ANY, PolicyNames=[])
```